app/views/govuk_publishing_components/audit/show.html.erb

```diff
@@ -2,284 +2,291 @@
 
 <%= render 'govuk_publishing_components/components/title', title: "Components audit", margin_top: 0; %>
 
-<% if ENV["MAIN_COMPONENT_GUIDE"] %>
-  <div class="govuk-tabs" data-module="govuk-tabs">
-    <h2 class="govuk-tabs__title">
-      Contents
-    </h2>
-    <ul class="govuk-tabs__list">
-      <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
-        <a class="govuk-tabs__tab" href="#applications">
-          Applications
-        </a>
-      </li>
-      <li class="govuk-tabs__list-item">
-        <a class="govuk-tabs__tab" href="#components-gem">
-          Components
-        </a>
-      </li>
-    </ul>
-    <div class="govuk-tabs__panel" id="applications">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: "Applications",
-        font_size: "l",
-        margin_bottom: 6
-      } %>
+<div class="govuk-tabs" data-module="govuk-tabs">
```

Comment:
Is there a reason for not using the component here?

Reply:
There was a reason for using not-the-component when the auditing was first built, I can't quite recall why, but that's no longer the case so have updated, also the details one. Shout if there's any others I've missed, thanks.